php/alpine-php7-min/Makefile

```diff
@@ -0,0 +1,27 @@
+NAME = phalconphp/php
+TAG = alpine-php7-min
```

Comment:
TAG `alpine-php7-min`
Folder `alpine-php7-min`
Dockerfile for `phalconphp/php:alpine-3-php7`